scene/gui/tree.cpp

```diff
@@ -5724,13 +5735,22 @@ bool Tree::is_recursive_folding_enabled() const {
 	return enable_recursive_folding;
 }
 
+void Tree::set_enable_drag_unfolding(bool p_enable) {
+	enable_drag_unfolding = p_enable;
```

Comment:
This should probably stop the Timer.
Unlikely that someone will run into this, but if you disable the property while hovering, it would still unfold.

Reply:
I'll check if it's enabled in the timer callback, that'll be nice and robust